index.js

```diff
@@ -1,13 +1,83 @@
 // TODO: Include packages needed for this application
+const inquirer = require("inquirer");
```

Comment:
good job importing all your files/packages at the top of the file - this is great for readability! :)